exercises/es6-classes/example-class.js

```diff
@@ -0,0 +1,22 @@
+// function ExampleClass() {
+
+// }
+
+class ExampleClass {
+  ExampleClass() {
+
+  }
```

Comment:
You can leave out this `ExampleClass()` method and just do `class ExampleClass {}`.